faucet.go

```diff
@@ -591,7 +547,7 @@ func main() {
 	mux.HandleFunc("/claim", createGetCoinsHandler(db))
 
 	c := cors.New(cors.Options{
-		AllowedOrigins:   []string{"https://faucet.terra.money", "http://localhost", "localhost", "http://localhost:3000", "http://localhost:8080"},
+		AllowedOrigins:   []string{"https://faucet.paloma.money", "http://localhost", "localhost", "http://localhost:3000", "http://localhost:8080"},
```

Comment:
we have domain `paloma.money`?

Reply:
Seems unlikely.

Reply:
we don't. Terra had a lot under their terra.money domai, not just the faucet. I think faucet.palomachain.com would work for us. - cc @taariq

Reply:
Let's encrypt?